include/mbedtls/pk.h

```diff
@@ -430,7 +430,8 @@ int mbedtls_pk_can_do( const mbedtls_pk_context *ctx, mbedtls_pk_type_t type );
  *
  * \return          1 if the context can do operations on the given type.
  * \return          0 if the context cannot do the operations on the given
- *                  type. This is always the case for a context that has
+ *                  type, or for non-allowed algorithms and usage flags.
```

Comment:
Wording: inserting this here makes the "This" on the next line weird. Either change the order to "0 for non-allowed [...] or if the context can't [...]. This is always the case [...]" or change the wording "0 if the context can't [...] or for non-allowed [...] or for a context that [...]".

(See also my other comment about this.)